docs/source/conf.py

```diff
@@ -0,0 +1,65 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../..'))
+sys.setrecursionlimit(1500)
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'pytorch-datastream'
+copyright = '2020, Richard Löwenström, Felix Abrahamsson, Jim Holmström'
+author = 'Richard Löwenström, Felix Abrahamsson, Jim Holmström'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
```

Comment:
this needs to follow the actual version of the lib later right?

Reply:
Good point. Not sure how we would update that automatically

Reply:
probably good ol' sed :D

Reply:
We'll see when we get there :)

Reply:
Tried to fix this